cuML/test/quasi_newton.cu

```diff
@@ -20,11 +21,14 @@ struct QuasiNewtonTest : ::testing::Test {
   const static double *nobptr;
   const static double tol;
   const static double X[N][D];
-  cublasHandle_t cublas;
+  cumlHandle *handle_ptr;
+  cumlHandle_t cuml_handle;
   cudaStream_t stream;
   void SetUp() {
-    cublasCreate(&cublas);
-    stream = 0;
+    cumlError_t status;
+    cumlCreate(&cuml_handle);
```

Comment:
Why us the C instead of the C++ API here? Using the C++ API would simplify the code.